Pilot: store latest push instead of global

howardjohn opened the pull request:
Attempt to fix https://github.com/istio/istio/issues/36791

istio-testing commented:
Skipping CI for Draft Pull Request.
If you want CI signal for your change, please convert it to an actual PR.
You can still manually trigger a test run with `/test all`

howardjohn commented:
/test all

istio-testing commented:
@howardjohn: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498461963708010496) | true | `/test integ-telemetry_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498461963708010496) | true | `/test integ-telemetry_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498461963708010496) | true | `/test integ-telemetry_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-security_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security_istio/1498461966920847360) | true | `/test integ-security_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498461963708010496) | true | `/test integ-telemetry_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-security_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security_istio/1498461966920847360) | true | `/test integ-security_istio`
integ-telemetry-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-mc_istio/1498461963749953536) | true | `/test integ-telemetry-mc_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/release-notes_istio/1498461972834816000) | true | `/test release-notes_istio`
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-cni_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-cni_istio/1498461963670261760) | true | `/test integ-cni_istio`
integ-distroless_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-distroless_istio/1498461963825451008) | true | `/test integ-distroless_istio`
integ-ipv6_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-ipv6_istio/1498461963859005440) | true | `/test integ-ipv6_istio`
integ-pilot_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot_istio/1498461963934502912) | true | `/test integ-pilot_istio`
integ-pilot-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote_istio/1498461965255708672) | true | `/test integ-pilot-istiodremote_istio`
integ-pilot-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-multicluster_istio/1498461964437819392) | true | `/test integ-pilot-multicluster_istio`
integ-pilot-istiodremote-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-pilot-istiodremote-mc_istio/1498461966086180864) | true | `/test integ-pilot-istiodremote-mc_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498461963708010496) | true | `/test integ-telemetry_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-security_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security_istio/1498461966920847360) | true | `/test integ-security_istio`
integ-telemetry-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-mc_istio/1498461963749953536) | true | `/test integ-telemetry-mc_istio`
integ-security-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-istiodremote_istio/1498461968615346176) | true | `/test integ-security-istiodremote_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-telemetry-istiodremote_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498461963787702272) | true | `/test integ-telemetry-istiodremote_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-telemetry-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-mc_istio/1498461963749953536) | true | `/test integ-telemetry-mc_istio`
integ-telemetry_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498803533674188800) | true | `/test integ-telemetry_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-telemetry-mc_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-mc_istio/1498461963749953536) | true | `/test integ-telemetry-mc_istio`
integ-telemetry_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498803533674188800) | true | `/test integ-telemetry_istio`
integ-telemetry-istiodremote_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498803533867126784) | true | `/test integ-telemetry-istiodremote_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@howardjohn: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-operator-controller_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-operator-controller_istio/1498461963896754176) | true | `/test integ-operator-controller_istio`
integ-security-multicluster_istio | 1651f1bc66bcb5e21982ad19d0ea28c95ed18f4e | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-security-multicluster_istio/1498461967759708160) | true | `/test integ-security-multicluster_istio`
integ-telemetry_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry_istio/1498803533674188800) | true | `/test integ-telemetry_istio`
integ-telemetry-istiodremote_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-istiodremote_istio/1498803533867126784) | true | `/test integ-telemetry-istiodremote_istio`
integ-telemetry-mc_istio | dc50480bf977dff9c95964c1cb71495191b7afb4 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/37604/integ-telemetry-mc_istio/1498803533787435008) | true | `/test integ-telemetry-mc_istio`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

howardjohn commented:
/test all

howardjohn commented:
cc @ramaraochavali @hzxuzhonghu WDYT? I cannot come up with any cases where it could break but want some more eyes

hzxuzhonghu commented on pilot/pkg/xds/ads.go:
```diff
@@ -650,6 +653,7 @@ func (s *DiscoveryServer) computeProxyState(proxy *model.Proxy, request *model.P
 	if gateway && proxy.Type == model.Router {
 		proxy.SetGatewaysForProxy(push)
 	}
+	proxy.LastPushContext = push
```
Can you move this line up?

	if push == nil {
		push = s.globalPushContext()
                proxy.LastPushContext = push
	}

hzxuzhonghu reviewed (commented):
LGTM

ramaraochavali commented:
Where we using globalContext() now after this change? Probably in some tests only? Should we cleanup that also?

howardjohn commented:
We use it when we initiate Pushes. Let me add comments and clean things up

On Thu, Mar 3, 2022, 3:36 AM Rama Chavali ***@***.***> wrote:

> Where we using globalContext() now after this change? Probably in some
> tests only? Should we cleanup that also?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/istio/istio/pull/37604#issuecomment-1057955480>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEYGXL3NLNDNRDVZMXGBR3U6CP4XANCNFSM5PSS7JYA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

howardjohn commented:
> Where we using globalContext() now after this change? Probably in some tests only? Should we cleanup that also?

Cleaned up the remaining incorrect usages (there are a small number of correct usages) and added comments

ramaraochavali reviewed (approved):
Thankyou.

howardjohn commented:
/retest
